pubspec.yaml

```diff
@@ -14,4 +14,4 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dev_dependencies:
-  test: ^1.11.1
+  test: '^1.11.1'
```

Comment:
It might be worthwhile to add a `CONTRIBUTING.md` that explains how to run the tests.

I was initially unaware that I had to run `flutter pub test` rather than `flutter test` to test this package.